scripts/spack/packages/axom/package.py

```diff
@@ -252,10 +258,12 @@ def initconfig_hardware_entries(self):
                 entries.append(
                     "# cuda_arch could not be determined\n\n")
 
-            if "+cpp14" in spec:
-                cudaflags += " -std=c++14"
-            else:
-                cudaflags += " -std=c++11"
+            if spec.satisfies("^blt@:0.6.1"):
```

Comment:
Not too clear on Spack syntax: does this specify a "BLT at least version 0.6.1" check, or is this checking for BLT with the exact version number? If it's the latter, is there a way of specifying the former?

Reply:
I think it means "If we're using a BLT up to and including 0.6.1, do the following"
(It is the `:` right after the the `@` that specifies the end of the range, but leaves the beginning of the range open).

Reply:
Ah gotcha, thanks!

Reply:
The syntax mimics the python list slice syntax and is inclusive of the number given.  Which can be a bit odd and sometimes you have to create a version that doesn't exist.